docs-src/python/how-to-set-workflow-retry-options-in-python.md

```diff
@@ -2,32 +2,20 @@
 id: how-to-set-workflow-retry-options-in-python
 title: How to set Workflow Retry Options in Python
 sidebar_label: Workflow Retry Options
-description: Set the Retry Policy from either start_workflow() or execute_workflow().
-tags:
-  - python
-  - how-to
+description: Set the Retry Policy to either start_workflow() or execute_workflow().
 ---
 
-Set the Retry Policy from either the [`start_workflow()`](https://python.temporal.io/temporalio.client.Client.html#start_workflow) or [`execute_workflow()`](https://python.temporal.io/temporalio.client.Client.html#execute_workflow) asynchronous methods.
+Set the Retry Policy to either the [`start_workflow()`](https://python.temporal.io/temporalio.client.Client.html#start_workflow) or [`execute_workflow()`](https://python.temporal.io/temporalio.client.Client.html#execute_workflow) asynchronous methods.
 
-```python
-handle = await client.start_workflow(
-    "your-workflow-name",
-    "some arg",
-    id="your-workflow-id",
-    task_queue="your-task-queue",
-    start_signal="your-signal-name",
-    retry_policy=RetryPolicy(maximum_interval=timedelta(seconds=2)),
-)
-```
+<a class="dacx-source-link" href="https://github.com/temporalio/documentation-samples-python/blob/main/workflow_timeouts_retries/workflows_dacx.py">View source code</a>
 
 ```python
-handle = await client.execute_workflow(
-    "your-workflow-name",
-    "some arg",
-    id="your-workflow-id",
-    task_queue="your-task-queue",
-    start_signal="your-signal-name",
-    retry_policy=RetryPolicy(maximum_interval=timedelta(seconds=2)),
-)
+# ...
+    handle = await client.execute_workflow(
+        YourWorkflow.run,
+        "your retry policy",
```

Comment:
Reading this string while browsing the docs, it is confusing to know what this string means. I might think it's some kind of "note" or "comment" for the workflow.